operation.go

```diff
@@ -392,6 +392,12 @@ func (operation *Operation) ParseParamComment(commentLine string, astFile *ast.F
 
 			param.Schema = schema
 		}
+	case "form":
```

Comment:
We have formData param type. Why do we need this ?

Reply:
The problem was caused by github.com/astaxie/beego v1.12.1

https://github.com/astaxie/beego/blob/de5650b7230083f957045890af942ffdac87c782/parser.go :377

Which I suppose we would be compatible to this situation.

Reply:
> We have formData param type. Why do we need this ?

The problem was caused by github.com/astaxie/beego v1.12.1

https://github.com/astaxie/beego/blob/de5650b7230083f957045890af942ffdac87c782/parser.go :377

Which I suppose swag would be compatible to this situation.

Reply:
@ubogdan hi, the problem was caused by swag meets github.com/astaxie/beego.

```text
ParseComment error in file /src/server/vendor/github.com/astaxie/beego/parser.go :form is not supported paramType
```

Reply:
We need a code example to understand the issue better.
After we agree with the changes, you will have to provide unit test for new change/functionality.

Reply:
This problem is caused by a upstream package, the swag program meet it will crashed. I suggest add this `form` type or skip the crash with an empty value.

:( `swag` program crashed that we can NOT update docs folder in my project.

Thanks.